code/modules/mob/living/carbon/human/death.dm

```diff
@@ -38,9 +38,10 @@
 	var/obj/item/organ/external/head = get_organ(BP_HEAD)
 	var/mob/living/simple_animal/borer/B
 
-	for(var/I in head.implants)
-		if(istype(I,/mob/living/simple_animal/borer))
-			B = I
+	if(head)
+		for(var/I in head.implants)
+			if(istype(I,/mob/living/simple_animal/borer))
+				B = I
 	if(B)
```

Comment:
Probably should be inside `if(head) {...}` block

Reply:
What if somebody finding a borer elsewhere?